src/SDKs/LabServices/LabServices.Tests/LabServices.Tests.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <Description>LabServices.Tests Class Library</Description>
+    <PackageId>LabServices.Tests</PackageId>
+    <AssemblyName>LabServices.Tests</AssemblyName>
+    <Version>1.0.0</Version>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
@ianhays for some reason it seems my previous comments were not posted.
Please remove line 8-14